Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN apt update \
 RUN apt install -y gcc g++ libgcc1 lib32gcc1 gdb libc6 git wget curl tar zip unzip binutils xz-utils liblzo2-2 cabextract iproute2 net-tools netcat telnet libatomic1 libsdl1.2debian libsdl2-2.0-0 \
     libfontconfig libicu63 icu-devtools libunwind8 libssl-dev sqlite3 libsqlite3-dev libmariadbclient-dev libduktape203 locales ffmpeg gnupg2 apt-transport-https software-properties-common ca-certificates tzdata
 
+## PAVLOV REQUIREMENT
+RUN apt install -y libc++-dev
```

Comment:
This doesn't have to be on a separate line with a game-specific comment. You could throw it into the existing apt install list above

Reply:
I just thought it'd be better so if someone doesn't need it, they can remove it on their side. It's only used by pavlov, and I haven't heard of any other games using it.

Reply:
Add it with the rest of the libs. It's fine.

Reply:
> Add it with the rest of the libs. It's fine.

Done :)